apis/nucleus/src/components/listbox/ListBox.jsx

```diff
@@ -132,9 +133,12 @@ export default function ListBox({
   }
   const isVertical = listLayout !== 'horizontal';
   const count = layout.qListObject.qSize.qcy;
-  const ITEM_SIZE = isVertical ? 33 : 200;
+  const SIZE_VERTICAL = 40;
+  const ITEM_SIZE = isVertical ? SIZE_VERTICAL : 200;
```

Comment:
It looks like ITEM_SIZE is changed from 33 to 44 even when not using checkboxes. why?

Reply:
`40px` is from the design I got. I can double-check just to make sure.

Reply:
I will take a broader look at the design in a subsequent PR when I have gotten more details. I think we can go with this for now.

Reply:
I think it should be an opt in to not affect any current use of nebula.js listbox out there

Reply:
Yes, you're right. We can keep it as it was while awaiting the precise design specs.

Reply:
@T-Wizard I have changed so that the larger row height (`40px`) is only used for checkboxes.
If the design suggests we should increase the height then we can bring up what you mentioned, whether this is a "breaking" change or not.